src/utils/async_helper.ts

```diff
@@ -0,0 +1,6 @@
+import history_file from '../../public/data/history.json';
+
+export async function load_history(skill: string, language: string): Promise<Record<string, Record<string, string>>> {
+  const history = (history_file as Record<string, any>)
+  return Promise.resolve(history[language][skill])
+}
```

Comment:
Does this really require async loading? AFAIK, json files can be read synchronously and using Astro imports you even get type safety out of the box: https://docs.astro.build/en/guides/imports/#json

Reply:
Yes, of course. This is just about the worst way of loading a static json-file there is. However, our plan was to setup a database at some point, that would supply this data through an Api. This way, whenever an event happens (for example the next German National Championship) you would not have to change the Website, only update the database and the content would automatically be generated. This function is written in a way that enables a drop-in replacement with an api call, that would have to be asynchronous.